public/docs/ts/_cache/tutorial/toh-pt5.jade

```diff
@@ -421,120 +503,171 @@ code-example(format='').
   ## Revise the *HeroDetailComponent*
 
   Before we rewrite the `HeroDetailComponent`, let's review what it looks like now:
+
 +makeExample('toh-4/ts/app/hero-detail.component.ts', null, 'app/hero-detail.component.ts (current)')
+
 :marked
-  The template won't change. We'll display a hero the same way. The big changes are driven by how we get the hero.
+  The template won't change. We'll display a hero the same way.
+  The big changes are driven by how we get the hero.
 
-  We will no longer receive the hero in a parent component property binding.
-  The new `HeroDetailComponent` should take the `id` parameter from the `params` observable
-  in the `ActivatedRoute` service and use the `HeroService` to fetch the hero with that `id`.
+block route-params
+  :marked
+    We will no longer receive the hero in a parent component property binding.
+    The new `HeroDetailComponent` should take the `id` parameter from the `params` observable
+    in the `ActivatedRoute` service and use the `HeroService` to fetch the hero with that `id`.
 
-  We need an import statement to reference the `ActivatedRoute` and `Params` types.
-+makeExample('toh-5/ts/app/hero-detail.component.ts', 'import-activated-route')(format=".")
-:marked
-  We import the `HeroService`so we can fetch a hero.
-+makeExample('toh-5/ts/app/hero-detail.component.ts', 'import-hero-service')(format=".")
 :marked
-  We import the `OnInit` interface because we'll call the `HeroService` inside the `ngOnInit` component lifecycle hook.
-+makeExample('toh-5/ts/app/hero-detail.component.ts', 'import-oninit')(format=".")
+  Add requisite imports first:
+
+- var _vers = _docsFor == 'dart' ? '' : '.1'
++makeExcerpt('app/hero-detail.component' + _vers + '.ts', 'added-imports', '')
+
+- var _ActivatedRoute = _docsFor == 'dart' ? 'RouteParams' : 'ActivatedRoute'
 :marked
-  We inject the both the `ActivatedRoute` service and the `HeroService` into the constructor as we've done before,
-  making private variables for both:
-+makeExample('toh-5/ts/app/hero-detail.component.ts', 'ctor', 'app/hero-detail.component.ts (constructor)')(format=".")
+  Let's have the `!{_ActivatedRoute}` service and the `HeroService` injected
```

Comment:
Let's get rid of the passive voice. how about:

have... injected -> inject...
constructor with...: -> constructor, saving their values in...: